"""Custom Exception Handlers."""

class AuthorizationError(Exception):
    """Catch and Throw Auth Errors."""

    pass



